test/hotspot/jtreg/compiler/rangechecks/TestRangeCheckHoistingScaledIV.java

```diff
@@ -30,6 +30,7 @@
  * @requires vm.flagless
  * @requires vm.debug & vm.compiler2.enabled
  * @requires os.simpleArch == "x64" | os.arch == "aarch64" | (os.arch == "riscv64" & vm.cpu.features ~= ".*rvv.*")
+ * @requires vm.opt.EnableVectorSupport == true
```

Comment:
BTW, I see more tests using jdk.incubator.vector besides of tests under test/hotspot/jtreg/compiler/vectorapi, like this test.
Could they also face the similar issue?